src/core/StelApp.cpp

```diff
@@ -563,12 +563,13 @@ void StelApp::init(QSettings* conf)
 
 	// Init audio manager
 	SplashScreen::showMessage(q_("Initializing audio..."));
```

Comment:
Hmm, probably these messages ("Initializing audio..." and "Initializing video...") shouldn't be displayed when features are suppressed...

Reply:
After another look, AudioMgr and VideoMgr are initialized with these messages even when MEDIA support is disabled. Just that the functions do nothing but write explanatory info to the logfile. The next message shows reasonably fast so that I don't think we need a change. The message helps to identify where a problem occurred.

Reply:
In one hand you are right, but in other hand we should suppressing these messages when media support is not exist in the binary package or one message when audio playback is disabled.